### PR TITLE
data: TRohit20

### PR DESCRIPTION
**Pull request by TRohit20**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

The community is awesome and all of you are doing an absolute great job. I would love to have my link free profile created and updated. Thank you :) 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5098"></a>

### CODE REVIEW

**Review by mahfuztbt — APPROVED** [review]
LGTM

**Review by shubhsharma19 — DISMISSED** [review]
LGTM 🚀

**Review by SurajPratap10 — APPROVED** [review]

File: data/TRohit20.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			"name": "Linkedin: You can Connect with me here",
+			"url": "https://www.linkedin.com/in/rohit-t-0124a4242/",
+			"icon": "FaLinkedin",
+			"color": "#00ACEE"
+		},
+		{
+			"group": "Primary Socials",
+			"name": "Github: You can Collaborate with me Or Check out my work",
+			"url": "https://www.linkedin.com/in/rohit-t-0124a4242/",
+			"icon": "FaLinkedin",
```
Comment (Sindhuinti) [inline data/TRohit20.json:51]: Hey you mention this as github but you are using Linkedin profile. Please once check this out

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/TRohit20.json (hunk 1 above)
Comment (TRohit20) [reply data/TRohit20.json]: Thank you for pointing it out @Sindhuinti . I just fixed it.

**Review by Sindhuinti — APPROVED** [review]
Thanks 🎉

**Review by mahfuztbt — APPROVED** [review]

**Review by SurajPratap10 — APPROVED** [review]

**Review by iamthecloverly — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**